build.gradle

```diff
@@ -373,3 +373,29 @@ jmh {
     iterations = 10
     fork = 2
 }
+
+configurations {
```

Comment:
could all this be extracted into a separate gradle file which we import? Similar to the eclipse stuff?